ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -345,6 +345,48 @@ defaults <- node_group <- node_group.node <- node
         inter_domain: < true | false | Default -> True >
 ```
 
+### BGP & MPLS-VPN Gateway
+
+```yaml
+< node_type_key >:
+
+  defaults:
+    # Node is acting as MPLS-VPN Gateway for EVPN to MPLS-VPN Interworking | Optional.
+    # A new BGP peer-group is used for this neighborship -> "bgp_peer_groups.mpls_vpn_gateway_overlay_peers".
+    # L3 Reachability is required for this to work, the preferred method to establish underlay connectivity is to use core_interfaces.
+    mpls_vpn_gateway:
+        enabled: < true | false | Default -> False >
+
+        # Domain IDs are required to perform D-Path lookups for loop prevention. If omitted, the defaults are used.
+        evpn_domain_id: < "nn:nn" | Default -> "65000:1" >
+        mpls_domain_id: < "nn:nn" | Default -> "65000:2" >
+
+        # D-path can be turned off for the inter-vpn export if desired.
+        enable_d_path: < true | false | Default -> true >
+
+        # Maximum number of routes to allow from the MPLS domain.
+        maximum_routes: < integer | Default -> 0 >
+
+        # Local-as to use when peering to the MPLS domain.
+        local_as: < bgp asn | Default -> False >
```

Comment:
We are trying to avoid mixing types in variables so it would be better to maybe not have a default value and if absent just not render it (I guess this is even maybe what the code is doing but I will check).

Additionally as a general comment on doc, the common format for variable name is to used somehting like `bgp_asn` rather than using spaces (same remark for the following variables)

Reply:
Changed doc to none instead of False. Also changed bgp asn to bgp_asn